fastlane/Fastfile

```diff
@@ -204,7 +204,7 @@ lane :test_carthage_integration do
   carthage_bootstrap
 
   scan(
-    project: "./Example/Carthage/ChatExample.xcodeproj",
+    project: "../Sample/Carthage/ChatExample.xcodeproj",
```

Comment:
Isn't the two `..` here a typo @buh ?

Reply:
It should be a single dot there, but it doesn't work, I want to see a new error message.

Reply:
It works locally with a single dot, but doesn't on GitHub.